temporal/t.rast.neighbors/testsuite/test_neighbors.py

```diff
@@ -5,21 +5,22 @@
 """
 
 import os
-import sys
-import unittest
 import grass.temporal as tgis
 from grass.gunittest.case import TestCase
 from grass.gunittest.gmodules import SimpleModule
 
 
-@unittest.skipIf(sys.version_info[0] > 2, "temporary disabled")
 class TestAggregationAbsolute(TestCase):
     @classmethod
     def setUpClass(cls):
         """Initiate the temporal GIS and set the region"""
         os.putenv("GRASS_OVERWRITE", "1")
         tgis.init()
         cls.use_temp_region()
+        cls.runModule("g.region", s=90, n=100, w=140, e=160, b=0, t=50, res=10, res3=10)
```

Comment:
```suggestion
        cls.runModule("g.region", s=0, n=90, w=140, e=160, b=0, t=50, res=10, res3=10)
```

Reply:
Not sure what exactly you wanted to test, so adjust as needed

Reply:
The point is to test that a map in the STRDS with a spatial extent outside the current computational region is not selected and processed if the _region_relation_ option is given....

Reply:
Yeah, I figured, the problem was n=100, but then I did not know what to put in s as it was 90

Reply:
Just to double check: Did the tests fail without the change above when you ran them locally? They pass in CI, but if they fail in your local run, I have to investigate...

Reply:
They failed locally in a latlong location because north was 100.

Reply:
Makes sense. Committed now.